dropshot_endpoint/src/lib.rs

```diff
@@ -150,6 +153,14 @@ fn do_endpoint(
         })
         .unwrap_or_default();
 
+    let visible = if let Some(true) = metadata.unpublished {
```

Comment:
Is there a test case around visible/unpublished?  (Sorry if I missed it!)

Reply:
There is. The openapi tests include an `unpublished` endpoint. We can tell they're successful because no change was required for the output json.

I wasn't sure about the visible vs. unpublished disparity, but I thought that `unpublished` was the right setting for endpoints, but it felt weird to carry the implicit negative into the struct; let me know if you have suggestions for clarity here.